### DISCUSSION TITLE
Save options of import / export

### DISCUSSION THREAD

**Discussion by GpxFeed** [devemux86/cruiser#49] [Q&A] [answered]
On testing Cruiser 3.0.9 - I made the wrong choice when importing GPX files - see "https://github.com/GpxFeed/" for an example.

Instead of selecting "Overlay" for the import, I mistakenly continued with the default setting "Routing".

For larger files, the import is very slow then and can only be stopped by exiting Cruiser.

Could Cruiser remember the last setting at this point as the default permanently?

Brinki

**Comment by devemux86** [comment]
@GpxFeed thanks for the suggestion!

Do you mean desktop Cruiser?

Import as overlay should be fast, as it just overlays the GPX content.
Import as routing could be slower, as it calls the selected router for a route calculation.
Route calculations on (external) routers are difficult to stop unless they fail or time out.

About your question, currently the import / export dialogs remember their settings while Cruiser is running.
I could save their settings locally, so Cruiser remembers them on restart?

**Reply by GpxFeed** [reply]
Yes, I mean the Desktop Cruiser - version 3.0.9 - I haven't tested the latest Android version yet.

The GPX import as overlay is indeed amazingly fast.

It would be fine if Cruiser would remember the last setting when restarting.

Brinki

**Comment by devemux86** [comment]
@GpxFeed nice POI collection!

We have probably discussed in the past? :slightly_smiling_face: 
It looks like you've been using desktop Cruiser for quite some time.

**Reply by GpxFeed** [reply]
Yes, there were a few mails.

My oldest Cruiser version was possibly 1.4.30 - cruiser-desktop-1.5.1 - Jan.12.2019 is still available as a backup.

As a replacement for the relevant, but not very complete campsite catalogues and internet services, I started at that time to prepare the OSM pois imported via https://overpass-turbo.eu per bash script for the existing map software in a usable way.

On the road and offline, Cruiser was used very successfully for trip planning - so far less for pure navigation.

I wrote about it - it would be nice if Cruiser would utilise the GPX **Link** tag. It is described in the German Wiki and "GPXSee" also uses it:
https://de.wikipedia.org/wiki/GPS_Exchange_Format

btw

The IDs **way** and **node** in the GPX Desc tag can be used to branch to further information in OSM - see:
https://www.flosm.de/html/POI-Karte.html

Brinki

**Reply by devemux86** [reply]
> it would be nice if Cruiser would utilise the GPX Link tag

This is already in my list for implementation. :slightly_smiling_face: 

> The IDs way and node in the GPX Desc tag can be used to branch to further information in OSM

Thanks for the interesting suggestions.

Can you please write them in new topics,
to discuss them better and not forget them here?

**Comment by GpxFeed** [comment]
#55

**Comment by devemux86** [comment]
@GpxFeed 

Improved import / export with save of their options will be available in the next Cruiser update.

**Comment by devemux86** [comment, marked as answer]
Save options of import / export is available in [Cruiser 3.0.16](https://play.google.com/store/apps/details?id=gr.talent.cruiser).

**Comment by devemux86** [comment]
Save options of import / export will also be available in the next **desktop** Cruiser update.
